zerver/lib/github.py

```diff
@@ -13,7 +13,9 @@ def get_latest_github_release_version_for_repo(repo: str) -> str:
     try:
         return requests.get(api_url).json()["tag_name"]
     except (requests.RequestException, json.JSONDecodeError, KeyError):
-        logger.error("Unable to fetch the latest release version from GitHub %s", api_url)
+        logger.exception(
+            "Unable to fetch the latest release version from GitHub %s", api_url, stack_info=True
```

Comment:
This will give two redundant stack traces, one from the exception, and the other from `stack_info`. We can omit `stack_info=True`.

Reply:
I added a bunch of `stack_info=True` in 2928bbc8bde7dcf1793a1f2beb7e8e81cd97c5e4, because missing it loses all stack information on the exception that was thrown in the `try` block.  In this case, there's not very much interesting that would be lost; but it seems like consistency in our `logging.exception` calls is worth it.

Reply:
Ah, I see, they’re not redundant. LGTM then.